### PR TITLE
Update Docker pages for 2.10.0

### PR DESCRIPTION
**Pull request by psiinon**
Most of these changes are already in the weeklies, and one of the
changes had already been added.

* Added -U user param + new scan hook calls
* Added GraphQL support
* Flagged -I as being post 2.9.0
* Added link to source folder
* Added link to example scan hooks
* Added note about docker networking

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

File: site/content/docs/docker/_index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ links:
 
   - name: 'API Scan'
     link: api-scan/
-    desc: a full scan of an API defined using OpenAPI / Swagger
+    desc: a full scan of an API defined using OpenAPI / Swagger or GraphQL (post 2.9.0)
```
Comment (kingthorin) [inline site/content/docs/docker/_index.md:20]:
```suggestion
    desc: a full scan of an API defined using OpenAPI / Swagger, or GraphQL (post 2.9.0)
```

Add a comma to make it just a bit more clear that the "(post 2.9.0)" only applies to GraphQL?

File: site/content/docs/docker/about.md [hunk 2]
```diff
@@ -35,6 +35,33 @@ The Dockerfiles can be found [here](https://github.com/zaproxy/zaproxy/tree/deve
 The docker file now supports [healthcheck](https://docs.docker.com/engine/reference/builder/#healthcheck). The check uses the `zap-cli status` to check that ZAP completed loading. If you are running ZAP with port other than the default `8080`, you need to set the `ZAP_PORT` environment variable. Otherwise, the healthcheck will fail.
 
 ## Usage Instructions:
+
+### Packaged Scans:
+All of the docker images provide a set of packaged scan scripts:
+
+* [Baseline Scan](../baseline-scan/) which runs the ZAP spider against the target for (by default) 1 minute followed by an optional ajax spider scan before reporting the results of the passive scanning.
+* [Full Scan](../full-scan/) which runs the ZAP spider against the target (by default with no time limit) followed by an optional ajax spider scan and then a full active scan before reporting the results.
+* [API Scan](../api-scan/) which performs an active scan against APIs defined by OpenAPI or GraphQL (post 2.9.0) via either a local file or a URL.
```
Comment (kingthorin) [inline site/content/docs/docker/about.md:44]:
```suggestion
* [API Scan](../api-scan/) which performs an active scan against APIs defined by OpenAPI, or GraphQL (post 2.9.0) via either a local file or a URL.
```

File: site/content/docs/docker/api-scan.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ type: docker
 
 The ZAP API scan is a script that is available in the ZAP [Docker](../about/) images.
 
-It is tuned for performing scans against APIs defined by OpenAPI via either a local file or a URL.
+It is tuned for performing scans against APIs defined by OpenAPI or GraphQL (post 2.9.0) via either a local file or a URL.
```
Comment (kingthorin) [inline site/content/docs/docker/api-scan.md:12]:
```suggestion
It is tuned for performing scans against APIs defined by OpenAPI, or GraphQL (post 2.9.0) via either a local file or a URL.
```

**Comment by psiinon** [comment]
Ah - I should have remembered the Oxford comma :D

**Comment by psiinon** [comment]
Done.

**Review by kingthorin — APPROVED** [review]

**Pull request merged**